src/main/java/com/ericsson/ei/jmespath/SplitFunction.java

```diff
@@ -0,0 +1,46 @@
+package com.ericsson.ei.jmespath;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+
+import io.burt.jmespath.Adapter;
+import io.burt.jmespath.JmesPathType;
+import io.burt.jmespath.function.ArgumentConstraints;
+import io.burt.jmespath.function.BaseFunction;
+import io.burt.jmespath.function.FunctionArgument;
+
+public class SplitFunction extends BaseFunction {
+
+    public SplitFunction() {
+        super(ArgumentConstraints.listOf(ArgumentConstraints.typeOf(JmesPathType.STRING),
+                ArgumentConstraints.typeOf(JmesPathType.STRING)));
+    }
+
+    public String[] split(String string, String separator) {
+        return string.split(separator);
+    }
+
+    @Override
+    protected <T> T callFunction(Adapter<T> runtime, List<FunctionArgument<T>> arguments) {
+        // TODO Auto-generated method stub
+        T value1 = arguments.get(0)
+                            .value();
+        T value2 = arguments.get(1)
```

Comment:
One of these values are the data and one of them are the seperator right? Then i think you should call them, data and seperator. Very unreadable otherwise.

Reply:
done